config/variables/stacks/back-office/training.hcl

```diff
@@ -46,8 +46,9 @@ locals {
     login_username = "pins-odt-sql-training-applications-bo"
     object_id      = "228592ab-d0f3-445b-b714-38148624c21f"
   }
-  feature_service_bus_enabled                        = "true"
-  notify_subscribers_function_gov_notify_template_id = "ee890e9f-5669-40ec-b3f1-f9e6f01024db"
+  feature_service_bus_enabled                              = "true"
+  notify_subscribers_function_gov_notify_template_id       = "ee890e9f-5669-40ec-b3f1-f9e6f01024db"

```

Comment:
Think ths is just following the existing pattern, but we need to think about training env as I'm not sure we should be using production Notify for it.
It's a bigger problem I think because the Node.env is set to production for the training env.